lib/src/styles/dimensions.dart

```diff
@@ -0,0 +1,8 @@
+abstract class Dimension {
```

Comment:
A better name would be `MovieDimensions` ?

Reply:
Agreed indeed. Fixed